cli/standalone/mod.rs

```diff
@@ -447,8 +450,9 @@ pub async fn run(
               CliNpmResolverManagedPackageJsonInstallerOption::ConditionalInstall(
                 package_json_deps_provider.clone(),
               ),
-            npm_registry_url,
             npm_system_info: Default::default(),
+            // TODO(bartlomieju): do we need to support it here?
+            npmrc: create_default_npmrc(),
```

Comment:
I don't think we need it here. The package information stored should be resolved based on the user's npmrc locally, but in the compiled executable it should know nothing about that. I think you can remove the todo and maybe add a comment that says the packages from different registries are already inlined or something.